src/main/java/io/github/thebusybiscuit/exoticgarden/ExoticGarden.java

```diff
@@ -106,11 +109,12 @@ public void onEnable() {
     }
 
     private void registerItems() {
-        mainCategory = new Category(new NamespacedKey(this, "plants_and_fruits"), new CustomItem(SkullItem.fromHash("a5a5c4a0a16dabc9b1ec72fc83e23ac15d0197de61b138babca7c8a29c820"), "&aExotic Garden - Plants and Fruits"));
-        miscCategory = new Category(new NamespacedKey(this, "misc"), new CustomItem(SkullItem.fromHash("606be2df2122344bda479feece365ee0e9d5da276afa0e8ce8d848f373dd131"), "&aExotic Garden - Ingredients and Tools"));
-        foodCategory = new Category(new NamespacedKey(this, "food"), new CustomItem(SkullItem.fromHash("a14216d10714082bbe3f412423e6b19232352f4d64f9aca3913cb46318d3ed"), "&aExotic Garden - Food"));
-        drinksCategory = new Category(new NamespacedKey(this, "drinks"), new CustomItem(SkullItem.fromHash("2a8f1f70e85825607d28edce1a2ad4506e732b4a5345a5ea6e807c4b313e88"), "&aExotic Garden - Drinks"));
-        magicalCategory = new Category(new NamespacedKey(this, "magical_crops"), new CustomItem(Material.BLAZE_POWDER, "&5Exotic Garden - Magical Plants"));
+        multiCategory = new MultiCategory(new NamespacedKey(this, "multi_category"), new CustomItem(Material.APPLE, "&aExotic Garden"));
```

Comment:
I think we should keep the nice textures for the parent category too.
So I would suggest to use the Dragon Fruit instead of an Apple here.

```suggestion
        multiCategory = new MultiCategory(new NamespacedKey(this, "parent_category"), new CustomItem(SkullItem.fromHash("847d73a91b52393f2c27e453fb89ab3d784054d414e390d58abd22512edd2b"), "&aExotic Garden"));
```